corehq/apps/hqwebapp/doc_lookup.py

```diff
@@ -47,6 +47,8 @@ def lookup_doc_id(doc_id):
     :param doc_id: ID of document to look for
     :returns: LookupResult or None
     """
+    if doc_id is None:
```

Comment:
`lookup_doc_id()` is called from three places, and all three (now) check that `doc_id` has a value, so currently this condition will never be true. But I added it to be explicit about the expected value, and to fail early in a way that would let future developers know immediately what the problem is.